doc/news/_preparation_next_release.md

```diff
@@ -388,9 +388,16 @@ removed due to:
 - New plugin to validate hex formatted colors (e.g. #fff or #abcd) and normalize them to rgba (4294967295 (= 0xffffffff) and 2864434397 (= 0xaabbccdd) respectively). It also has support for named colors according to the [extended color keywords](https://www.w3.org/TR/css-color-3/#svg-color) from CSS3.
   _(Philipp Gackstatter)_
 
+<<<<<<< HEAD
```

Comment:
Looks like you had a merge conflict and forgot to remove the [conflict markers](https://wincent.com/wiki/Understanding_Git_conflict_markers):

```suggestion

```

.

Reply:
Oh, sorry, I thought I removed the markers, but it seems like I didn't...